doc.html

```diff
@@ -47,104 +47,79 @@ <h2>4 produce_demand_forecaster Usage Scenarios</h2>
-
-For each parameter with available data at the specified boundary, a time series per aggregation is returned. Note that null values may be returned as part of a time series when no measurements are available.
-
-Therefore, those three parameters should be provided in the request, as it is shown below:
-<pre>
-+--------------------------------------------+
-| data                                       |
-+-------------+------------------------------+
-|      Inputs | Boundary                     |
-|             | Time interval                |
-|             | Required parameters          |
-+-------------+------------------------------+
-|     Outputs | Sensor data available        |
-+-------------+------------------------------+
-</pre>
-On the other hand, the capabilities endpoint contains all the parameters available for each produce_demand_forecaster service implementor. It also contains the aggregations defined for each parameter.
+The produce_demand_forecaster serves the forecast over the **forecast** endpoint.
 
-A query to the capabilities endpoint returns an array of strings representing the parameters supported by the implementor, given the availability of physical sensors within the requested boundaries.
 <pre>
 +--------------------------------------------+
-| capabilities                               |
+| forecast                                   |
 +-------------+------------------------------+
-|      Inputs | Boundary                     |
+|      Inputs | Time interval                |
+|             | 3-letter GEO code            |
 +-------------+------------------------------+
-|     Outputs | Supported parameters         |
+|     Outputs | array of top produce         |
+|             | (meta)data                   |
 +-------------+------------------------------+
 </pre>
-Please refer to the openAPI produce_demand_forecaster specification for details about data schemas and formats returned by the capabilities endpoint, as well as to ANNEX 1 for a list of normalized measurements that may be offered by a produce_demand_forecaster service.
-
-produce_demand_forecaster services are not required to handle intense traffic from a single client, such as the one that may result from being directly invoked on user interface interactions in an FMIS, for instance. Implementors MAY generate a 429 TOO MANY REQUESTS error response if the rate of calls exceeds some pre-defined quota. Clients requiring more intense field information are advised to implement a mirroring mechanism using the subscription functions.
-
-<h2>5.1 Queries with no data available</h2>
 
-It could happen that there is no data available for the requested parameters at the specified boundary and time interval. A common example for that could be that sensors for the requested parameters were never installed within the boundaries, or they were not online during the time interval.
+<h2>5.1 Invalid GEO code</h2>
```

Comment:
No sub-sections for errors. Keep in mind that most errors are better described in the OpenAPI document. Only document special cases here that may be hard to summarize in a short OAS description